### ISSUE TITLE
ie 11 Object.values not supported error

### ISSUE THREAD

**Issue by p51i** [phase2/particle#417]
v10.0.0. I have been getting a JS error on IE 11 only and after much research, narrowed it down to an unexpected lack of support for Object.values. It turns out that babel-polyfill does not transpile this method because it is evidently still just a "proposal." I am likely not properly describing, but suffice to say, my particle theme was breaking in [ie 11](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values).

 There appear to be a number of different fixes and workarounds for this issue, including an [npm package](https://www.npmjs.com/package/object.values).

To get my site (which is in production) working, I opted to add the following line to design-system.js, just above exports: `if (!Object.values)
  Object.values = obj => Object.keys(obj).map(key => obj[key]);`

I wanted to report this for 2 reasons: 1. I figured you might want to know, and 2. I wanted to get your feedback on my fix. If there is a better way to handle, I'm all ears.

**Comment by illepic** [comment]
Hey @p51i ! We ran into this a bit ago! We actually modified the webpack approach in 10.1 (releasing today) to ensure that `@babel/polyfill` is automatically included before **all** entry points: https://github.com/phase2/particle/blob/master/particle.js

The quick fix for you is to `npm install --save-dev @babel/polyfill` and make sure it's `import @babel/polyfill;` at the top of your application's entry point.

I had made the assumption too that Babel would handle that kind of polyfilling, but - NOPE! - we have to make sure `@babel/polyfill` is imported first.
